tools/requirements.txt

```diff
@@ -1,4 +1,3 @@
 lxml
 requests
-json
```

Comment:
Is there a special reason, why you deleted the json package from the requirements.txt?

Reply:
Yeah, because it's a built-in module. It gave me an error during install.

Note: Some distros / packages might not include it. 

https://stackoverflow.com/questions/41466431/pip-install-json-fails-on-ubuntu

I can gladly revert that commit if you prefer.

Reply:
No, it's all right. I just wondered why you removed it.